### DISCUSSION TITLE
Remove All problematic flags at once

### DISCUSSION THREAD

**Discussion by sunil-saini** [imapsync/imapsync#269] [General]
Hi Gilles LAMIRAL,

while doing imapsync we are getting flags errors - 

`msg INBOX/OUTBOX/669 {304236} could not append ( Subject:[MASKED], Date:["MASKED"], Size:[304236], Flags:[\Indexed \Seen] ) to folder OUTBOX: Error sending '503 APPEND OUTBOX (\Indexed \Seen) "MASKED" {304236}': 503 BAD Error in IMAP command APPEND: Invalid system flag \INDEXED (0.001 + 0.000 secs)`

`msg INBOX/17440 {51576} could not append ( Subject:[MASKED], Date:["MASKED"], Size:[51576], Flags:[\Seen \X-EON-HAS-ATTACHMENT] ) to folder INBOX: Error sending '33 APPEND INBOX (\Seen \X-EON-HAS-ATTACHMENT) "MASKED" {51576...AD Error in IMAP command APPEND: Invalid system flag \X-EON-HAS-ATTACHMENT (0.001 + 0.000 secs).`

So we added  **--regexflag 's/\\FORWARDED|\\Indexed|\\ATTACHED|\\RECEIPTCHECKED//gi'** parameter.

But as these un supported flags keep coming and it's not possible to know them all beforehand. So is there any way that we can specify that other than these few flags ( like \Answered \Flagged \Deleted \Seen \Draft $NotJunk ) , ignore all flags while syncing.

**Comment by gilleslamiral** [comment]
Thanks!
It's more an issue than a discussion, I can't add labels, users usually don't go here to solve issues.
Can you put it in the issues tab? I'll then reply and delete this. 
See you soon!

**Reply by sunil-saini** [reply]
Hey, have added in issues.
Thanks
